README.md

```diff
@@ -111,14 +111,36 @@ The screenshot method arguments:
         - Parameters:
 
           `Top` (int): Top coordinate of the ignore region.
-
           `Bottom` (int): Bottom coordinate of the ignore region.
-
           `Left` (int): Left coordinate of the ignore region.
-
           `Right` (int): Right coordinate of the ignore region.
         - Raises:IllegalArgumentException: If top, bottom, left, or right is less than 0 or top is greater than or equal to bottom or left is greater than or equal to right.
         - valid: Ignore region should be within the boundaries of the screen.
+
+## Running with Hybrid Apps
+
+For a hybrid app, we need to switch to native context before taking screenshot.
+
+- Add a helper method similar to following for say flutter based hybrid app:
+```java
+public void percyScreenshotFlutter(String name, ScreenshotOptions options) {
+    // switch to native context
+    driver.context("NATIVE_APP");
+    percy.screenshot(name, options);
```

Comment:
this does not show where this percy/driver is, you will need to pass it if this is a helper. which makes sense as user wouldnt want to copy this in every selenium file

Reply:
added